website/data/api-navigation.js

```diff
@@ -52,6 +52,7 @@ export default [
         ]
       },
       { category: 'nomad' },
+	  { category: 'openldap' },
```

Comment:
Nit, seems like the left brace is aligned vertically for the rest of the items here.